dev/TeachingTip/TeachingTip.cpp

```diff
@@ -1062,6 +1063,27 @@ void TeachingTip::OnActionButtonClicked(const winrt::IInspectable&, const winrt:
 
 void TeachingTip::OnPopupOpened(const winrt::IInspectable&, const winrt::IInspectable&)
 {
+    if (winrt::IUIElement10 uiElement10 = *this)
+    {
+        if (auto xamlRoot = uiElement10.XamlRoot())
+        {
+            m_currentXamlRootSize = xamlRoot.Size();
+            if (m_xamlRootChangedToken)
+            {
+                xamlRoot.Changed(m_xamlRootChangedToken);
+            }
+            m_xamlRootChangedToken = xamlRoot.Changed({ this, &TeachingTip::XamlRootChanged });
+        }
```

Comment:
is there not a winrt::auto_revoke version of this? If not, let's write a helper like we did for DP changed.